src/library/trackset/crate/cratestorage.cpp

```diff
@@ -24,44 +20,41 @@ const QString CRATESUMMARY_TRACK_COUNT = "track_count";
 const QString CRATESUMMARY_TRACK_DURATION = "track_duration";
 
 const QString kCrateTracksJoin = QString(
-        "LEFT JOIN %3 ON %3.%4=%1.%2").arg(
-                CRATE_TABLE,
-                CRATETABLE_ID,
-                CRATE_TRACKS_TABLE,
-                CRATETRACKSTABLE_CRATEID);
-
-const QString kLibraryTracksJoin = kCrateTracksJoin + QString(
-        " LEFT JOIN %3 ON %3.%4=%1.%2").arg(
-                CRATE_TRACKS_TABLE,
-                CRATETRACKSTABLE_TRACKID,
-                LIBRARY_TABLE,
-                LIBRARYTABLE_ID);
+        "LEFT JOIN %3 ON %3.%4=%1.%2")
+                                         .arg(
+                                                 CRATE_TABLE,
+                                                 CRATETABLE_ID,
+                                                 CRATE_TRACKS_TABLE,
+                                                 CRATETRACKSTABLE_CRATEID);
+
+const QString kLibraryTracksJoin = kCrateTracksJoin + QString(" LEFT JOIN %3 ON %3.%4=%1.%2").arg(CRATE_TRACKS_TABLE, CRATETRACKSTABLE_TRACKID, LIBRARY_TABLE, LIBRARYTABLE_ID);
 
 const QString kCrateSummaryViewSelect = QString(
         "SELECT %1.*,"
-            "COUNT(CASE %2.%4 WHEN 0 THEN 1 ELSE NULL END) AS %5,"
-            "SUM(CASE %2.%4 WHEN 0 THEN %2.%3 ELSE 0 END) AS %6 "
-            "FROM %1").arg(
-                CRATE_TABLE,
-                LIBRARY_TABLE,
-                LIBRARYTABLE_DURATION,
-                LIBRARYTABLE_MIXXXDELETED,
-                CRATESUMMARY_TRACK_COUNT,
-                CRATESUMMARY_TRACK_DURATION);
+        "COUNT(CASE %2.%4 WHEN 0 THEN 1 ELSE NULL END) AS %5,"
+        "SUM(CASE %2.%4 WHEN 0 THEN %2.%3 ELSE 0 END) AS %6 "
+        "FROM %1")
+                                                .arg(
+                                                        CRATE_TABLE,
+                                                        LIBRARY_TABLE,
+                                                        LIBRARYTABLE_DURATION,
+                                                        LIBRARYTABLE_MIXXXDELETED,
+                                                        CRATESUMMARY_TRACK_COUNT,
+                                                        CRATESUMMARY_TRACK_DURATION);
```

Comment:
this also looks strange

Reply:
wait, you can comment on multiple lines? This is amazing, didn't know that :o